docs/src/main/sphinx/connector/delta-lake.rst

```diff
@@ -438,6 +445,30 @@ The following example uses all three table properties::
   )
   AS SELECT name, comment, regionkey FROM tpch.tiny.nation;
 
+.. _renaming-tables:
+
+Renaming tables
+^^^^^^^^^^^^^^^
+
+When using Hive metastore, you can use the connector to :doc:`rename
+</sql/alter-table>` Delta Lake tables.
+
+Rename operations distinguish between:
+
+* Tables created without explicit location
+
+  Renaming will move the data on the underlying storage system. As this
+  can be an expensive operation, an explicit opt-in is required. Set
+  ``delta.rename-managed-tables.enabled`` to ``true`` in your catalog
+  properties and restart your Trino server to activate this behavior.
```

Comment:
an explicit opt-in is required with setting ``delta.rename-managed-tables.enabled`` to ``true``. 

Drop the rest